app/angular/src/client/preview/angular/components/app.component.ts

```diff
@@ -2,7 +2,7 @@
 // to provide @Inputs and subscribe to @Outputs, see
 // https://github.com/angular/angular/issues/15360
 // For the time being, the ViewContainerRef approach works pretty well.
-
+import * as _ from 'lodash';
```

Comment:
Are we ok with adding the full `lodash` here to the bundle?

Reply:
It's strange thing, i have measured just now and bundle with separate imports is bigger than with  full lodash, so i'm not sure
```
import {  has,  get, set,  isUndefined,  isFunction,  invoke,  isEmpty,  forEach} from 'lodash';
My bundle size --> 5950355
```
```
import * as _ from 'lodash';
My bundle size --> 5950264
```

Reply:
In this case, there is no difference between importing with `*` or with the named import, since it's transpiled to `require('lodash')` in the end, and lodash is not tree-shakable (I think we even don't tree shake anything in storybook). 
I think that the file size is bigger because of the extra generated code per each method. 

I did [this](https://github.com/storybooks/storybook/pull/2558) PR to change lodash to lodash-es. And [here](https://medium.com/@martin_hotell/tree-shake-lodash-with-webpack-jest-and-typescript-2734fa13b5cd) also a nice read about it.

Reply:
That's really helpful thank you!